### ISSUE TITLE
expandable sender/recipient sections are skipped in tab order

### ISSUE THREAD

**Issue by ganthern** [tutao/tutanota#4650]
**Bug in web app**

**Describe the bug**
expanding sender/recipient section in multi-column only works with keyboard if previously clicked, not via tabbing
same for multiple attachment expander

**To Reproduce**
1. Go to mail view and select a mail
2. try to focus the recipient or attachment expander with tab
4. that doesn't work

** Additional Context **
when clicking the expander with the mouse, you can expand / collapse them with enter normally

**Comment by charlag** [comment]
The whole section with expanding should be in the tab order? Need to re-check

**Comment by rezbyte** [comment]
This appears to be fixed already in the latest master.
